### DISCUSSION TITLE
SDK support for Playwright-Python

### DISCUSSION THREAD

**Discussion by WinnieLi12345** [percy/cli#815] [Ideas]
Hi,

I knew there is a SDK for percy-playwright with nodejs.
I'm wondering if it is possible to support percy-playwright with python in the future :)

**Comment by Robdel12** [comment]
Hey @WinnieLi12345! It's not too likely this will happen any time soon (unless this discussion gets a lot of attention). 

In the mean time, we have an extensive build your own SDK doc that could help unstick you: https://docs.percy.io/docs/build-your-own-sdk

**Reply by WinnieLi12345** [reply]
Thank you! I will try it.
